save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -123,7 +128,14 @@ class DockerService(private val configProperties: ConfigProperties) {
             ClassPathResource(SAVE_CLI_EXECUTABLE_NAME).inputStream,
             File(resourcesPath, SAVE_CLI_EXECUTABLE_NAME)
         )
+        val baseImage = when (sdk) {
+            is Jdk -> "openjdk:${sdk.version}"
```

Comment:
Please test the logging if the version that you have provided is missing.
Otherwise, if the logging is bad, please handle the exception and write a proper log message